remote_it/pom.xml

```diff
@@ -214,6 +218,8 @@
                                 <exclude>com/splicemachine/mrio/**</exclude>
                                 <exclude>**/CallableTransactionIT.java</exclude>
                                 <exclude>**/VacuumIT.java</exclude>
+                                <!-- EE ITs failing in remote mode -->
```

Comment:
Do we have a jira to track all those disabled tests?

Reply:
No specific JIRA. Currently, it would be too generic without a specific plan what to do about them. Most failing tests were developed to rely on files in the local file system. We can't enable that with just a JDBC connection and no access to remote HDFS.